lib/src/repo.rs

```diff
@@ -134,16 +134,16 @@ impl ReadonlyRepo {
             .insert(store.root_commit_id().clone());
 
         let op_heads_path = repo_path.join("op_heads");
-        fs::create_dir(&op_heads_path).unwrap();
+        fs::create_dir(&op_heads_path)?;
```

Comment:
I think by doing this, we lose stack trace information, i.e. which operation specifically failed, compared to panicking (from which the line number should be recoverable?). It would be best to introduce a new error type and wrap these steps.

Reply:
Replaced with an error that carries the path with it for context.